File: array/withFunc.java
package springIntro;

import java.util.*;

public class withFunc {

	static int elements = 15;
	
	static void takeNotes(int notes[]) {
		int pass = 0, stay = 0;
		for(int i = 0; i < elements; i++) {
			if(notes[i] > 50) {
				pass++;
			}
			else {
				stay++;
			}
			System.out.print(notes[i] + " ");
		}
		System.out.printf("\nPass = %d\nStay = %d", pass, stay);
	}
	
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int notes[] = new int[elements];
		System.out.println("Enter notes");
		for(int i = 0; i < elements; i++) {
			notes[i] = in.nextInt();
		}
		takeNotes(notes);
	}

}

File: array/withFunc.cpp
#include<iostream>
using namespace std;
static int elements = 15;
void takeNotes(int notes[]){
	int pass = 0, stay = 0;
	cout << "Notes = ";
	for(int i = 0; i < elements; i++){
		if(notes[i] < 5){
			stay++;
		}
		else{
			pass++;
		}
		cout << notes[i] << " ";
	}
	cout << "\nPass = " << pass << endl;
	cout << "Stay = " << stay << endl;
}
int main(){
	int notes[elements];
	cout << "Enter notes" << endl;
	for(int i = 0; i < elements; i++){
		cin >> notes[i];
	}
	takeNotes(notes);
	return 0;
}
